project/src/main/world/creature/creature.gd

```diff
@@ -39,8 +39,11 @@ export (bool) var suppress_sfx: bool = false setget set_suppress_sfx
 # how high the creature's torso is from the floor, such as when they're sitting on a stool or standing up
 export (int) var elevation: int setget set_elevation
 
+export (ChatIcon.BubbleType) var chat_bubble_type: int setget set_chat_bubble_type, get_chat_bubble_type
```

Comment:
document how this `chat_bubble_type` field is a virtual field, and does not hold a meaningful value

Reply:
Added `virtual property; value is only exposed through getters/setters` comment here, and throughout the project in a few other places